app/models/Service.class.test.js

```diff
@@ -26,7 +26,21 @@ describe('Service model from service raw data', () => {
     expect(serviceModel.merchantDetails.addressLine2).to.equal('10 Downing Street')
     expect(serviceModel.merchantDetails.city).to.equal('London')
     expect(serviceModel.merchantDetails.postcode).to.equal('AW1H 9UX')
-    expect(serviceModel.merchantDetails.countryName).to.equal('GB')
+    expect(serviceModel.merchantDetails.countryName).to.equal('United Kingdom')
+  })
+
+  it('when the contry code is not recognised then it should return undefined', () => {
+    const serviceModel = new Service(serviceFixtures.validServiceResponse({
+      organisationName: 'Give Me Your Money',
+      merchant_details: {
+        address_line1: 'Clive House',
+        address_line2: '10 Downing Street',
+        address_city: 'London',
+        address_postcode: 'AW1H 9UX'
+      }
```

Comment:
Could we add a test for if there is a country code that will not map to anything (`ZZ` should work because the ISO have said that will never be assigned) just to check the behaviour is correct in this case?

Reply:
Good catch.  

The pay-js-commons function did not handle this gracefully.
So updated it in pay-js-commons and then bumped ths version of the NPM module in this PR.